version: "3.5"
services:
  nextcloudpi:
    container_name: ncp_container
    hostname: nextcloud
    command: "${NEXTCLOUD_URL}"
    # ports:
    #   - "80:80"
    #   - "443:443"
    #   - "4443:4443"
    volumes:
      - "${DATA_DISK_PATH}:/data"
      - "${BACKUP_DISK_PATH}:/backup"
    # Needed due to error with access to the system clock (https://unix.stackexchange.com/q/598471)
    privileged: true
    
    restart: unless-stopped
    image: ownyourbits/nextcloudpi:old
    networks:
      - my_network

networks:
  my_network:
    name: "my_network"
    external: ${CONNECT_TO_EXTERNAL_DOCKER_NETWORK}
